packages/bsync/proto/bsync.proto

```diff
@@ -41,6 +43,38 @@ message ScanMuteOperationsResponse {
   string cursor = 2;
 }
 
+message NotifOperation {
+  enum Setting {
+    SETTING_UNSPECIFIED = 0;
+    SETTING_UNFILTERED = 1;
+    SETTING_PRIORITY = 2;
+  }
+  string id = 1;
+  string actor_did = 2;
+  Setting setting = 3;
```

Comment:
Considering giving this a different name since there could be other notification settings we want to track.